apk-generator/v2/app/static/js/main.js

```diff
@@ -225,6 +233,7 @@ $form.submit(function (e) {
     data.append("email", $emailInput.val());
     data.append("data-source", dataSourceType);
     data.append("build-type", buildType);
+    data.append("is-auth-enabled", isAuthEnabled);
```

Comment:
@niranjan94 `$authOptionCheckbox.is(":checked")` also working fine :smile: 
Thanks @iamareebjamal @niranjan94